contrib_bots/lib/xkcd.py

```diff
@@ -0,0 +1,139 @@
+from random import randint
+
+import logging
+import requests
+
+XKCD_TEMPLATE_URL = 'http://xkcd.com/%s/info.0.json'
+LATEST_XKCD_URL = 'http://xkcd.com/info.0.json'
+
+
```

Comment:
Alright, I just trying to follow PEP8 style guide that recommends two blank lines beetween top level functions and class definitions, should I delete it ?

https://www.python.org/dev/peps/pep-0008/#blank-lines